packages/react/src/addresses/hooks/__tests__/__fixtures__/Adresses.fixtures.tsx

```diff
@@ -11,9 +11,9 @@ import {
 import React, { Fragment } from 'react';
 import useAddresses from '../../useAddresses';
 
-const booleanToText = boolean => (boolean ? 'yes' : 'no');
+const booleanToText = (boolean:boolean) => (boolean ? 'yes' : 'no');
 
-export const Addresses = ({ userId }) => {
+export const Addresses = ({ userId }: {userId:number}) => {
```

Comment:
I would just create a type, to be more readable. (and I don't know why, but it has no spaces between the names.)

```suggestion
export const Addresses = ({ userId }: Props) => {
```

and then: 

```
type Props = {
  userId: number;
}
```

and btw, down below, in this file, there is this line:  

`} = useAddresses({ userId });`

this should have an error in your code, because that hook was changed (I guess I missed it when I changed). The new correct code should be like this:

`} = useAddresses(false, userId);`